packages/goto/src/index.js

```diff
@@ -193,14 +202,19 @@ module.exports = ({ defaultDevice = 'Macbook Pro 13', timeout, ...deviceOpts })
       ...args
     }
   ) => {
+    const prePromises = []
+
     if (adblock) {
-      await engine.enableBlockingInPage(page)
+      prePromises.push(engine.enableBlockingInPage(page))
```

Comment:
@remusao do you think we should force await?

Reply:
tested, it works fine 🙂

Reply:
Looks good to me!